samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -7,6 +7,9 @@ var gltfPipeline = require('gltf-pipeline');
 var path = require('path');
```

Comment:
The tileset.json asset version string should be `"1.1"` when `--3d-tiles-next` is set.

Reply:
fixed